app/(authenticated)/comic-issue/[id]/upload-stateful-covers/page.tsx

```diff
@@ -66,21 +64,33 @@ export default function UploadComicIssueStatefulCoversPage({ params }: { params:
 
 		const formData = new FormData()
 
-		let i = 0
-		for (const cover of issueCovers) {
-			if (cover.image) formData.append(`covers[${i}][image]`, cover.image)
-			formData.append(`covers[${i}][artist]`, cover.artist)
-			formData.append(`covers[${i}][isSigned]`, cover.isSigned.toString())
-			formData.append(`covers[${i}][isUsed]`, cover.isUsed.toString())
-			formData.append(`covers[${i}][rarity]`, cover.rarity)
-			i = i + 1
-		}
-
-		await updateStatelessCovers(formData)
+		await Promise.all(
+			issueCovers.map(async (cover, index) => {
+				if (cover.image) {
+					const imagesToMerge = [{ src: cover.image }]
+					if (cover.isUsed) imagesToMerge.push({ src: usedOverlayImage.src })
+					else imagesToMerge.push({ src: unusedOverlayImage.src })
+
+					if (cover.isSigned && comicIssue.signature) imagesToMerge.push({ src: comicIssue.signature })
+
+					const mergedImageDataURL = await mergeImages(imagesToMerge, { crossOrigin: 'anonymous' })
```

Comment:
wondering if we should put dpublisher origin as a value for `crossOrigin` instead of `anonymous`?
@d-reader-josip thoughts?